src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2137,9 +2144,9 @@ static void _cloneNode(SvgNode* from, SvgNode* parent)
 }
 
 
-static void _postponeCloneNode(SvgLoaderData* loader, SvgNode *node, char* id)
+static void _postponeCloneNode(Array<SvgNodeIdPair>* nodes, SvgNode *node, char* id)
```

Comment:
Suggestion: ```Array <SvgNodeIdPair>& nodes```

Reply:
may I ask why? so that in the future I'll use the preferred method right away

I didn't apply this suggestion, since this function was defined earlier and it is directly connected with this pr (I needed to predefine it), though I can do it in a separate pr in all places (clonePostponedNodec, stylePostponedNodes, postponeNodeClone)